jumanji/environments/routing/cleaner/env.py

```diff
@@ -98,8 +97,10 @@ def __init__(
             render_mode: the mode for visualising the environment, can be "human" or "rgb_array".
             penalty_per_timestep: the penalty returned at each timestep in the reward.
         """
-        self.num_agents = num_agents
-        self.generator = generator or RandomGenerator(num_rows=10, num_cols=10)
+        self.generator = generator or RandomGenerator(
+            num_rows=10, num_cols=10, num_agents=3
```

Comment:
You have to update the docstring to reflect that the default generator takes `num_agents=3`, right?